sys-fs/btrfs-progs/btrfs-progs-4.3.1.ebuild

```diff
@@ -72,10 +82,36 @@ src_configure() {
 }
 
 src_compile() {
-	emake V=1
+	local static=""
+
+	if use static || use static-libs; then
+		static="static"
```

Comment:
I don't think there is any reason to build the "static" target when USE="static-libs". The ebuild already builds libbtrfs.a anyway.